lightningd/plugin.c

```diff
@@ -284,6 +284,8 @@ static const char *plugin_log_handle(struct plugin *plugin,
 		level = LOG_UNUSUAL;
 	else if (json_tok_streq(plugin->buffer, leveltok, "error"))
 		level = LOG_BROKEN;
+	else if (json_tok_streq(plugin->buffer, leveltok, "io"))
+		level = LOG_IO_IN;
```

Comment:
These levels are special, you need to call log_io() not log_.  And you need a direction, so you should probably make the strings 'io-in' and 'io-out'?